src/ai-mate/README.md

```diff
@@ -0,0 +1,24 @@
+# Symfony AI - Mate Component
+
+The Mate component provides an MCP (Model Context Protocol) server that enables AI assistants to interact with Symfony applications through standardized tools.
```

Comment:
we should add the development purpose here as well i feel

Reply:
@Nyholm i have rewritten it a bit - maybe you can also go over it